src/encoder/encodermp3.cpp

```diff
@@ -254,23 +254,21 @@ void EncoderMp3::setEncoderSettings(const EncoderSettings& settings)
     int modeoption = settings.getSelectedOption(EncoderMp3Settings::ENCODING_MODE_GROUP);
     m_encoding_mode = (modeoption==0) ? vbr_off : (modeoption==1) ? vbr_abr : vbr_default;
 
-    if (m_encoding_mode != vbr_off) {
-        if (lame_set_VBR_quality) {
-            // vbr range 0 to 9.999 (best to worst). slider range (0 to max) worse to best.
-            int val = settings.getQualityValues().size() - 1 - settings.getQualityIndex();
-            if (val < 10) {
-                m_vbr_index = val;
-            } else {
-                m_vbr_index = 9.999f;
-            }
+    if (m_encoding_mode == vbr_off) {
+        if (m_bitrate > 100 ) {
```

Comment:
This requires some comments in the code. 
Do you have a reference why the threshold for mono is 100?